exercises/practice/tournament/.docs/instructions.append.md

```diff
@@ -0,0 +1,25 @@
+# dummy
+
+## jq-specific Instructions
```

Comment:
Is there any particular reason to diverge from the problem spec here and expect a different output than the formatted string?

Reply:
I guess it's not terrible to expect the canonical result: change the last line from
```
| @csv
```
to 
```
| tabular
```
and the tabular function is
```
def padright(n): tostring + (" " * n) | .[:n];
def padleft(n):  (" " * n) + tostring | .[-n:];

def tabular:
  [.[0] | padright(30)] + [.[1:][] | padleft(2)]
  | join(" | ")
;
```

Do you think doing some string formatting is better value than finding out about builtin output formatters?

Reply:
I'm inclined to stick to the problem specs. The CSV output is good to know about but if we want to push students in that direction, perhaps that should be done via a concept exercise.